google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -338,7 +338,13 @@ private static void ensureKmsKeyRingIamPermissionsForTests(
             .build();
     requestParamsHeader.put(requestParamsKey, "parent=" + kmsKeyRingResourcePath);
     iamStub = MetadataUtils.attachHeaders(iamStub, requestParamsHeader);
-    iamStub.setIamPolicy(setIamPolicyRequest);
+    try {
```

Comment:
Could you add minimum permissions to run integration tests in CONTRIBUTION documentation in a subsequent PR?

Reply:
@frankyn, I mentioned IT_SERVICE_ACCOUNT_EMAIL env var which is required for one IT test.